### ISSUE TITLE
Login with keypass storage doesn't work for new setups

### ISSUE THREAD

**Issue by markrietveld** [Netflix-Skunkworks/go-jira#93]
This may be the same as #82 , but that ticket doesn't have enough details to be sure.

Since https://github.com/Netflix-Skunkworks/go-jira/commit/85298e9, the keypass login method only works if you're already logged in. [This line](https://github.com/Netflix-Skunkworks/go-jira/commit/85298e9#diff-7963896e2d02a84fc72737d56d0caf39R19) makes it so that if the password isn't already there, it will fail the command, instead of asking for the password [here](https://github.com/Netflix-Skunkworks/go-jira/commit/85298e9#diff-7963896e2d02a84fc72737d56d0caf39R40).

This leads to:
```
./jira-darwin-10.6-amd64 login
panic: keyring: Password not found
```

**Comment by jamiesonio** [comment]
I moved back to Version 0.1.9 in order to workaround setting up the keychain the first time, but after moving to latest version 0.1.14 I am still seeing this issue.

**Comment by wrouesnel** [comment]
This should be resolved by 8d8167b02df92903ca0d6b9515059e3299efc1b9 FYI. I'm using it successfully now.

**Comment by coryb** [comment]
Closing, I think this was fixed in #104 as indicated.
